From 3441893e62c078a962f51bd058fbf19adaa234a5 Mon Sep 17 00:00:00 2001
From: Moritz Lammerich <moritz.lammerich@ost.ch>
Date: Wed, 14 Oct 2020 15:26:54 +0200
Subject: [PATCH] remove cflags

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47777f4..0ad5c91 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,11 +1,6 @@
 cmake_minimum_required(VERSION 2.6)
 project(flinklib)
 
-## Compiler flags
-if(CMAKE_COMPILER_IS_GNUCC)
-	set(CMAKE_C_FLAGS "-std=gnu99")
-endif()
-
 ## Set library version
 include(cmake/versionCheckHelpers.cmake)
 setLibraryVersion()
