diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ece461..cd6dc3b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,9 +29,9 @@ INSTALL(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/includes/eeros DESTINATION include
 
 ## Install CMake configuration file
 set(INCLUDE_INSTALL_DIR "include/")
-set(LIBUCL_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/libucl-src/include")
+# set(LIBUCL_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/libucl-src/include")
 set(LIB_INSTALL_DIR lib/)
-set(LIBUCL_LINK_DIR "${CMAKE_CURRENT_BINARY_DIR}/libucl-build/")
+# set(LIBUCL_LINK_DIR "${CMAKE_CURRENT_BINARY_DIR}/libucl-build/")
 
 include(CMakePackageConfigHelpers)
 
@@ -39,7 +39,7 @@ configure_package_config_file(
 	cmake/EEROSConfig.cmake.in
 	${CMAKE_CURRENT_BINARY_DIR}/EEROSConfig.cmake
 	INSTALL_DESTINATION ${LIB_INSTALL_DIR}/EEROS/cmake
-	PATH_VARS INCLUDE_INSTALL_DIR LIBUCL_INCLUDE_DIR LIB_INSTALL_DIR LIBUCL_LINK_DIR)
+	PATH_VARS INCLUDE_INSTALL_DIR LIB_INSTALL_DIR)
 
 write_basic_package_version_file(
 	${CMAKE_CURRENT_BINARY_DIR}/EEROSConfigVersion.cmake
diff --git a/cmake/EEROSConfig.cmake.in b/cmake/EEROSConfig.cmake.in
index 4c3e774..7b73448 100644
--- a/cmake/EEROSConfig.cmake.in
+++ b/cmake/EEROSConfig.cmake.in
@@ -1,6 +1,4 @@
 @PACKAGE_INIT@
 
 set_and_check(EEROS_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
-set_and_check(EEROS_LIBUCL_INCLUDE_DIR "@PACKAGE_LIBUCL_INCLUDE_DIR@")
 set_and_check(EEROS_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
-set_and_check(EEROS_LIBUCL_LINK_DIR "@PACKAGE_LIBUCL_LINK_DIR@")
