diff --git a/examples/Makefile b/examples/Makefile
index 2052fb1..a7d8b87 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -20,8 +20,8 @@ LINKER		:= gcc
 WFLAGS		:= -Wall -Wextra -Werror=float-equal -Wuninitialized \
 	-Wunused-variable -Wdouble-promotion -pedantic -Wmissing-prototypes \
 	-Wmissing-declarations -Werror=undef
-CFLAGS		:= -g -pthread -I $(INCLUDEDIR)
-LDFLAGS		:= -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
+CFLAGS		:= $(EXTRA_CFLAGS) -g -pthread -I $(INCLUDEDIR)
+LDFLAGS		:= $(EXTRA_LDFLAGS) -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
 
 # commands
 RM		:= rm -rf
diff --git a/library/Makefile b/library/Makefile
index df460f2..ee89e38 100644
--- a/library/Makefile
+++ b/library/Makefile
@@ -22,9 +22,9 @@ LINKER		:= gcc
 WFLAGS		:= -Wall -Wextra -Werror=float-equal -Wuninitialized \
 	-Wunused-variable -Wdouble-promotion -pedantic -Wmissing-prototypes \
 	-Wmissing-declarations -Werror=undef
-CFLAGS		:= -g -fPIC -I $(INCLUDEDIR)
+CFLAGS		:= $(EXTRA_CLFAGS) -g -fPIC -I $(INCLUDEDIR)
 OPT_FLAGS	:= -O1
-LDFLAGS		:= -lm -lrt -pthread -shared -Wl,-soname,$(SONAME)
+LDFLAGS		:= $(EXTRA_LDFLAGS) -lm -lrt -pthread -shared -Wl,-soname,$(SONAME)
 
 # different compile flags for math libs
 MATH_OPT_FLAGS	:= -O3 -ffast-math -ftree-vectorize
diff --git a/services/rc_battery_monitor/Makefile b/services/rc_battery_monitor/Makefile
index d63eb71..a842033 100644
--- a/services/rc_battery_monitor/Makefile
+++ b/services/rc_battery_monitor/Makefile
@@ -19,8 +19,8 @@ LINKER		:= gcc
 
 # compiler and linker flags
 WFLAGS		:= -Wall -Wextra -Werror=float-equal -Wuninitialized -Wunused-variable -Wdouble-promotion
-CFLAGS		:= -g -pthread -I $(INCLUDEDIR)
-LDFLAGS		:= -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
+CFLAGS		:= $(EXTRA_CFLAGS) -g -pthread -I $(INCLUDEDIR)
+LDFLAGS		:= $(EXTRA_LDFLAGS) -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
 
 # commands
 RM		:= rm -rf
diff --git a/services/robotcontrol/Makefile b/services/robotcontrol/Makefile
index 6c85cd9..d6bf1f0 100644
--- a/services/robotcontrol/Makefile
+++ b/services/robotcontrol/Makefile
@@ -19,8 +19,8 @@ LINKER		:= gcc
 
 # compiler and linker flags
 WFLAGS		:= -Wall -Wextra -Werror=float-equal -Wuninitialized -Wunused-variable -Wdouble-promotion
-CFLAGS		:= -g -pthread -I $(INCLUDEDIR)
-LDFLAGS		:= -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
+CFLAGS		:= $(EXTRA_CFLAGS) -g -pthread -I $(INCLUDEDIR)
+LDFLAGS		:= $(EXTRA_LDFLAGS) -lm -lrt -pthread -L $(LIBDIR) -l:librobotcontrol.so.1
 
 # commands
 RM		:= rm -rf
